sdk/schemaregistry/azure-schemaregistry-avroserializer/samples/eventhub_send_integration.py

```diff
@@ -59,10 +59,9 @@ def send_event_data_batch(producer, serializer):
 
 
 # create a AvroSerializer instance
-# TODO: after 'azure-schemaregistry==1.0.0b3' is released, update 'endpoint' to 'fully_qualified_namespace'
```

Comment:
is there other TODOs left in the code base?

Reply:
no I just looked for them and removed all of them in the tests/samples/code

Reply:
will check one more time after merging